### DISCUSSION TITLE
Instalar o restadataware 1.14 no delphi 12

### DISCUSSION THREAD

**Discussion by joaovictorsyspan** [OpenSourceCommunityBrasil/REST-DataWare#392] [General]
Bom dia, uso a versão 14.1 e preciso instalar ele no Delphi 12, sei que existe uma versão mais nova, mas agora não consigo atualizar devido a tempo. Quando compilar no Delphi 12 da um erro, alguem consegui instalar. Obrigado

**Comment by mobius1qwe** [comment]
Na capa do repositório, acesse o grupo do telegram do componente já que ninguém do projeto lê as discussões aqui.
